trustlessness.adoc

```diff
@@ -222,29 +222,29 @@ disarmament. https://twitter.com/Truthcoin/status/1491415722123153408?s=20&t=ZyR
 switched it around to highlight the rejection of trust and the
 importance of running a full node].
 
-It's up to the users to decide to what degree they want to verify the
+It's up to the users deciding to what degree they want to verify the
```

Comment:
This smells a bit weird to me. Could you please make sure that the sentence is correct. I really can't say :smile: